style.css

```diff
@@ -0,0 +1,59 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;500;700&display=swap");
+
+body {
+  background: white;
```

Comment:
- [ ] You should use the background color for the body as provided in the Figma design. (`#E5E5E5`;)

Reply:
Your background is still white. Kindly adjust according to Figma design specification and as specified by previous reviewer